frontend/utilities/helpers.ts

```diff
@@ -617,6 +617,31 @@ export const humanHostDetailUpdated = (detailUpdated?: string): string => {
   }
 };
 
+const DISK_ENCRYPTION_MESSAGES = {
+  darwin: {
+    enabled:
+      "The disk is encrypted. The user must enter their<br/> password when they start their computer.",
+    disabled:
+      "The disk might be encrypted, but FileVault is off. The<br/> disk can be accessed without entering a password.",
+  },
+  windows: {
+    enabled:
+      "The disk is encrypted. If recently turned on,<br/> encryption could take awhile.",
+    disabled: "The disk is unencrypted.",
+  },
+};
+
+export const humanHostDiskEncryptionEnabled = (
+  platform?: string,
```

Comment:
can this be a narrower type? think it may be time to add something like

```ts
type IPlatforms = "darwin" | "windows" | "linux"
```

Reply:
I tried, but since the backend is string, I ran into a rabbit hole of types. I think it can also be "rhel"?

Reply:
ok no worries 👍🏽